### PR TITLE
add docs/provisioners/helm.md

### PR DESCRIPTION
**Pull request by akihikokuroda**
Part of #522

Signed-off-by: akihikokuroda <akuroda@us.ibm.com>

### CODE REVIEW

File: docs/provisioners/helm.md [hunk 1]
```diff
@@ -73,7 +83,7 @@ will be updated to Unpacked, indicating that all its contents have been stored o
 ```console
 $ kubectl get bundle -l app=my-ahoy
 NAME                 TYPE   PHASE      AGE
-my-ahoy-5764594dc8   http   Unpacked   33s
+my-ahoy-5764594dc8   git    Unpacked   33s
```
Comment (tylerslaton) [inline docs/provisioners/helm.md:86]: Should this actually be `http`? The BD above this is using `http` and not `git`.

File: docs/provisioners/helm.md [hunk 2]
```diff
@@ -194,31 +230,96 @@ $ kubectl get deployments.apps my-ahoy-hello-world -o yaml | grep 'image:' | xar
...
 
-The values file for the chart can be applied during install.  The values can be embeded in the `config` section of BundleDeployment.
+The name of the deployment is `my-ahoy-fromvalues` now
 
-```yaml
+### Update the values file
+
+Change this a value in the values file.  Update the `nameOverride` to `name1`
+
+
```
Comment (tylerslaton) [inline docs/provisioners/helm.md:320]: nit:
```suggestion
```

File: docs/provisioners/helm.md [hunk 3]
```diff
@@ -256,9 +357,138 @@ spec:
...
+helm list
+```
+
+```console
+NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
+my-ahoy rukpak-system   3               2022-08-27 22:12:51.056782394 +0000 UTC deployed        hello-world-0.1.1       1.16.0
+```
+
+Now the helm chart version is `hello-world-0.1.1`.
+
```
Comment (tylerslaton) [inline docs/provisioners/helm.md:470]: nit:
```suggestion
```

**Review by tylerslaton — COMMENTED** [review]
Looks good! Couple of small things and I'll be ready to approve.

File: docs/provisioners/helm.md (hunk 1 above)
Comment (akihikokuroda) [reply docs/provisioners/helm.md:86]: Yes, that's right.  I edited some bundles to use http.  I changed this by mistake, too.  Thanks!

File: docs/provisioners/helm.md (hunk 3 above)
Comment (akihikokuroda) [reply docs/provisioners/helm.md:470]: Thanks!

**Comment by akihikokuroda** [comment]
Put into #552

**Comment by akihikokuroda** [comment]
Put into #552

**Pull request closed without merging**